examples/demo-jobs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM neuromation/base:v1.6
+FROM neuromation/base:v1.7.8
```

Comment:
I'm pretty sure that the dependabot can check/upgrade this tag automatically.
Would be nice to set this up in a separate PR.